llvm/unittests/ADT/CountCopyAndMove.h

```diff
@@ -12,15 +12,16 @@
 namespace llvm {
 
 struct CountCopyAndMove {
+  static int Constructions;
   static int CopyConstructions;
   static int CopyAssignments;
   static int MoveConstructions;
   static int MoveAssignments;
   static int Destructions;
   int val;
 
-  CountCopyAndMove() = default;
-  explicit CountCopyAndMove(int val) : val(val) {}
+  CountCopyAndMove() { ++Constructions; }
```

Comment:
think there's a better name we could come up with for this? I worry it'd confuse users who might think this is all constructions (including copy and move) when it isn't...

Could split it into DefaultConstructions and ValueConstructions, maybe?

Reply:
Sorry for the delay, yeah that works just fine, updated the names 👍